dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -490,15 +491,46 @@ export const renderElement = ({
 				</Island>
 			);
 		case 'model.dotcomrendering.pageElements.MediaAtomBlockElement':
-			return (
-				<VideoAtom
-					format={format}
-					assets={element.assets}
-					poster={element.posterImage?.[0]?.url}
-					caption={element.title}
-					isMainMedia={isMainMedia}
-				/>
-			);
+			/*
+				- MediaAtomBlockElement is used for self-hosted videos
+				- Historically, these videos have been self-hosted for legal or sensitive reasons
+					- These videos play in the `VideoAtom` component
+				- Looping videos, introduced in July 2025, are also self-hosted
+					- Thus they are delivered as a MediaAtomBlockElement
+					- However they need to display in a different video player
+				- We need to differentiate between the two forms of video
+					- We can do this by interrogating the atom's metadata, which includes the new attribute `videoPlayerFormat`
+
+				- Note: we'll probably extend this functionality to handle new 'Cinemagraph' videos
+					- These may use the looping video, or yet another new, video player
```

Comment:
You should be able to pass in `videoStyle="Cinemagraph"` [here](https://github.com/guardian/dotcom-rendering/pull/14843/files#diff-c6430efd1bc9eb99ffd16bb8e72143998e693c0d0ee84fa5e2ac22b7ed460a2fR40) and it'll just work.

Reply:
Great, thanks. I've [removed this comment](https://github.com/guardian/dotcom-rendering/pull/14843/commits/08835beb32da0183fa2f2497b7ce89b005f581d5) now the path forward is clear.